### DISCUSSION TITLE
How to identify new devices

### DISCUSSION THREAD

**Discussion by hocomadvies** [Koenkk/zigbee2mqtt#5492] [Devices]
Hi I have a bunch of temp / humidity sensors implementing them is no problem. But after that how to see which sensor is which? On the sensor there is a serial number. But as far as I can see it is not in the log / data. How are you dealing with this? It is not always possible iclude one and make a note per device and then to the next...

**Comment by DrTron** [comment]
When I include several devices of the same type, I label them first (1-x) and include them in that order. Then the first new device in devices.yaml is No. 1 and so forth.

To identify devices I would try to trigger them and see what comes up in the logs. Close a contact sensor, hold a temperature sensor in your hand to warm it up, etc...

**Reply by hocomadvies** [reply]
Thanks for your reply. The triggering thing is what I already do. I was hoping there would be a way to identify the devices with the serial number. That would be easy. Numbering them is a good idea too i will see if that works for me.

**Reply by DrTron** [reply]
For obvious reasons, I then include that number in the friendly name, too. Saves a lot of headache.
All Zigbee devices do have a unique identifier (which is what comes up first as the friendly name when pairing), but it's usually not printed on the device. Probably too much hassle during production.

**Reply by hocomadvies** [reply]
Ok thanks :)
